qiskit/result/mitigation/local_readout_mitigator.py

```diff
@@ -214,7 +217,7 @@ def quasi_probabilities(
             probs_dict[index] = probs_vec[index]
 
         quasi_dist = QuasiDistribution(
-            probs_dict, stddev_upper_bound=self.stddev_upper_bound(shots, qubits)
+            probs_dict, shots=shots, stddev_upper_bound=self.stddev_upper_bound(shots, qubits)
```

Comment:
why do you need to add `shots=shots` here?

Reply:
The `quasi_dist` is returned from the method, and I would like the number of shots the distribution was derived from to be part of the object. Currently I have in my code: 
```
...
mitigated_quasi_probs = mitigator.quasi_probabilities(noisy_counts)
mitigated_quasi_probs.shots=number_of_shots # required, since 
...
```
This PR allows:
```
...
mitigated_quasi_probs = mitigator.quasi_probabilities(noisy_counts, shots=number_of_shots)
...
```
In general the number of shots should be added to results if the information is available.